### DISCUSSION TITLE
Unable to save packets

### DISCUSSION THREAD

**Discussion by Kinglion17** [dannagle/PacketSender#356] [Q&A]
<img width="369" alt="packetsender" src="https://github.com/dannagle/PacketSender/assets/157514394/0e4bfbd4-1753-469c-a1b2-43076c6cdc5a">

I installed Packet Sender Version : 8.6.5 but nothing happens after clicking on "save". All fields are filled i.e Name, ASCII, port , Resend delay and the protocol type.

**Comment by dannagle** [comment]
Is it possible it got saved but buried by the all the bundled example packets?

Ideas...
- Try searching for it. 
- Try saving it by putting a 0 in the front such as `0 my packet` to make it sort at the top.
- Final attempt.. you could try the portable version that saves everything to the local directory.

Hope one of those helps.

**Reply by Kinglion17** [reply]
Thanks

**Comment by Kinglion17** [comment]
Interesting. With the old version, I'm able to save my packets and see displayed on the dashboard but not for the 8.6.5.

**Reply by dannagle** [reply]
Does the portable version work as expected?

**Reply by Kinglion17** [reply]
I think re-downloaded the new portable version and extracted to the downloads folder . I'm now able to save the packets after running packet sender from the downloads folder. I wonder if exporting the files to the program folder might have been the issue. Question - when downloading the zip file, do we also need to download the installer ?

**Reply by dannagle** [reply]
The portable zipped version is completely self-contained. You do not need anything else.
